lightpath/ui/widgets.py

```diff
@@ -120,13 +120,16 @@ class LightRow(InactiveRow):
     parent : QObject, optional
     """
     MAX_HINTS = 2
+    device_updated = pyqtSignal()
```

Comment:
maybe switch it to use `qtpy` and `Signal` instead of `pyqtSignal` ?

Reply:
https://github.com/pcdshub/lightpath/blob/415410dfa810af825337580c437409b51b6a416d/lightpath/ui/widgets.py#L9

Reply:
I guess this is how Teddy updated to qtpy. Should I take the opportunity to make it clearer by removing the as import?

Reply:
It seems like a good idea!